msi/rust.wxs

```diff
@@ -0,0 +1,87 @@
+        <Package
+            Comments="Rust is a systems programming language that runs blazingly fast, prevents almost all crashes, and eliminates data races."
+            InstallerVersion="200"
+            Compressed="yes" />
+
+        <Icon Id="rust.ico" SourceFile="rust-logo.ico" />
+        <Property Id="ARPPRODUCTICON" Value="rust.ico" />
+        <!-- txt format is not supported -->
+        <WixVariable Id="WixUILicenseRtf" Value="LICENSE.rtf" />
+
+        <!--
+            Source media for the installation.
+            Specifies a single cab file to be embedded in the installer's .msi.
+        -->
+        <Media Id="1" Cabinet="contents.cab" EmbedCab="yes" CompressionLevel="high" />
+        <!-- <Media Id="1" Layout="Files" /> -->
+
+        <!-- Installation directory and files are defined in Files.wxs -->
+        <Directory Id="TARGETDIR" Name="SourceDir">
+            <Directory Id="ProgramFilesFolder">
+                <Directory Id="INSTALLDIR" Name="Rust">
+                    <!-- Root directories for every feature should have different IDs for correct work heat.exe -->
+                    <Directory Id="Rustc" Name="." />
+                    <Directory Id="Gcc" Name="." />
+                    <Directory Id="Docs" Name="." />
+                    <Directory Id="Cargo" Name="." />
+                </Directory>
+            </Directory>
+
+            <Component Id="PathEnv" Guid="*">
+                <RegistryValue Root="HKLM" Key="Software\[Manufacturer]\[ProductName]" Name="InstallDir" Type="string" Value="[INSTALLDIR]" KeyPath="yes" />
+                <!-- Slash is NOT needed in "[INSTALLDIR]bin" -->
+                <Environment Id="PATH" Name="PATH" Value="[INSTALLDIR]bin" Permanent="no" Part="last" Action="set" System="yes" />
+            </Component>
+
+        </Directory>
+
+        <Feature Id="Rustc"
+                 Title="Rust compiler and standard crates"
+                 Display="0"
+                 Level="1">
+                 <ComponentGroupRef Id="RustcGroup" />
+        </Feature>
+        <Feature Id="Gcc"
+                 Title="Linker and platform libraries"
+                 Display="1"
+                 Level="1">
+                 <ComponentGroupRef Id="GccGroup" />
+        </Feature>
+        <Feature Id="Docs"
+                 Title="HTML documentation"
+                 Display="2"
+                 Level="1">
+                 <ComponentGroupRef Id="DocsGroup" />
+        </Feature>
+        <Feature Id="Cargo"
+                 Title="Cargo, the Rust package manager"
+                 Display="3"
+                 Level="1">
+                 <ComponentGroupRef Id="CargoGroup" />
+        </Feature>
+        <Feature Id="Path"
+                 Title="Add to PATH"
+                 Description="Add Rust to PATH environment variable"
+                 Display="4"
+                 Level="5">
+                 <ComponentRef Id="PathEnv" />
+        </Feature>
+
```

Comment:
We should set `AllowAdvertise=false` for all features,- demand install does not make sense for Rust.